src/components/ResultsMap.vue

```diff
@@ -50,7 +50,14 @@ export default {
 				};
 			}
 		},
-		handleMarkerClick(marker) {
+		resetCenter() {
```

Comment:
Okay, so at first the map is centered based on the user's geolocation.  Then, when the user clicks on a marker, the coordinates are reset.  I'm wondering how a latitude of 0 and a longitude of 0 affect the centering on the map.  Could you clarify why `0` has been set as the value for both?

Reply:
I have the same question, particularly since 0, 0 is a legit set of coords that land us in the Atlantic Ocean.

Reply:
This is definitely an interesting solution. Does that make the map snap at all before it resets with the proper coordinates?